modules/installation-initializing.adoc

```diff
@@ -225,15 +225,26 @@ ifndef::aws,gcp[]
 * Retrieve a {op-system-first} image and upload it to an accessible location.
 endif::aws,gcp[]
 endif::restricted[]
-ifndef::nutanix[]
-* Obtain service principal permissions at the subscription level.
-endif::nutanix[]
+ifdef::azure[]
+* You have an Azure subscription ID and tenant ID. For more information, see "Recording the subscription and tenant ID" in "Configuring an Azure account".
+* If you are installing the cluster using a service principal, you have its application ID and password. For more information, see "Creating a service principal" in "Configuring an Azure account".
+* If you are installing the cluster using a system-assigned managed identity, you have enabled it on the virtual machine that you will run the installation program from.
+* If you are installing the cluster using a user-assigned managed identity, you have met these prerequisites:
+** You have its client ID.
+** You have assigned it to the virtual machine that you will run the installation program from.
+endif::azure[]
```

Comment:
Worth noting that //https://github.com/openshift/openshift-docs/pull/37275 previously added the service principal req specifically for Azure, but did not add the azure attribute. As a result, the req appeared in all install use cases using this module. Correcting this oversight as part of osdocs-5126.